src/app/debug/table/table.component.ts

```diff
@@ -350,14 +386,14 @@ export class TableComponent implements OnInit, OnDestroy {
     }
   }
 
-  getStatusColor(metadata: any): string {
-    let statusName = this.viewSettings.currentView.metadataNames.find((name: string) => {
+  getStatusColor(metadata: DebugListItem): string {
```

Comment:
Right now it's doing an unnecessary find for an attribute that matches the string `status`, this can be removed:
```ts
    getStatusColor(metadata: any): string {
    if (metadata.debugVariables.status) {
      const status = metadata.debugVariables.status;
      if (status.toLowerCase() === 'success') {
        return '#c3e6cb';
      } else if (status.toLowerCase() === 'null') {
        return '#A9A9A9FF';
      } else {
        return '#f79c9c';
      }
    }
    return 'none';
  }
```